Update handling-relations.md with an example of ReferenceArrayInput (#1144)
api-platform/docs: admin/handling-relations.md

    >
      <AutocompleteInput optionText="title" />
    </ReferenceInput>
    // or
    <ReferenceInput 

>         source="book.@id" 

Reviewer: I think using the `format` prop is clearer than having this source.

```

The autocomplete field should now work properly!

> Let's say `useEmbedded` parameter is set to `true` and you have nested a nested objects array you want to use in an ReferenceArrayInput. ReferenceArrayInput expects an array of ids. In this case you have to tell it, where it finds those ids. This is done via "format"

Reviewer: ```suggestion
Let's say `useEmbedded` parameter is set to `true` and you have nested an array of objects you want to use in a `ReferenceArrayInput`. `ReferenceArrayInput` expects an array of ids. In this case, you have to tell it where to find those ids. This is done by using the `format` prop.
```

> ```javascript

Reviewer: Maybe it's better to add this example above than to write again the example.

import React from "react";
import {
  HydraAdmin,
  ResourceGuesser,
  CreateGuesser,
  EditGuesser,
  InputGuesser
} from "@api-platform/admin";
import { ReferenceInput, AutocompleteInput } from "react-admin";

...
const BooksEdit = props => (
  <EditGuesser {...props}>
    ...
    

>     <ReferenceArrayInput source="books" reference="books_reference" label="Books" sort={{field: 'title', order: 'ASC'}}

Reviewer: Could you indent properly?

>                   format={v => (v ? v.map(i => (i["@id"] 11? i["@id"] : i)) : [])}>

Reviewer: ```suggestion
                  format={v => (v ? v.map(i => (i["@id"] || i))) : []}>
```
